tests/rtt_ros2_params_tests/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+cmake_minimum_required(VERSION 3.5)
+project(rtt_ros2_params_tests)
+
+# Skip building this package if BUILD_TESTING is OFF
+find_package(ament_cmake REQUIRED)
+if(NOT BUILD_TESTING)
+  ament_package()
+  return()
+endif()
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+#   add_compile_options(-Wall -Wextra -Wpedantic)
+# endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rtt_ros2 REQUIRED)
+find_package(rtt_ros2_node REQUIRED)
+find_package(rtt_ros2_params REQUIRED)
+# find_package(rtt_ros2_test_msgs REQUIRED)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+endif()
+
+ament_package()
+
+# unit tests
+find_package(ament_cmake_gtest REQUIRED)
+
+# The test_ros_params fails with
+#
+#     double free or corruption (fasttop)
+#
+# during destruction in ROS dashing. This seems to be a known problem:
+# https://gitlab.com/ApexAI/performance_test/-/issues/87
+# ==> workaround: disable the test for dashing
+#
+set(ROS_DISTRO $ENV{ROS_DISTRO})
+if(ROS_DISTRO STREQUAL "dashing")
+  return()
+endif()
```

Comment:
This obviously has been copied from [rtt_ros2_topics_tests/CMakeLists.txt](https://github.com/orocos/rtt_ros2_integration/blob/32e2c94ac0888ce255b2b507cdc38fbcb823952b/tests/rtt_ros2_topics_tests/CMakeLists.txt#L41-L52). Is it really the case here, too? If not, please remove this workaround for `rtt_ros2_params_tests`.